docs/zh/UserGuide/Ecosystem Integration/Prometheus.md

```diff
@@ -0,0 +1,74 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+## ����
+* ֧�ֵ�IoTDB�汾 : V0.12+
+* ֧�ֵ�Prometheus�汾 : V2.23+
+* ֧�ֵ�Go�汾 : V1.13+
+
+## ��IoTDB
+
+�μ�https://iotdb.apache.org/zh/UserGuide/Master/Get%20Started/QuickStart.html

```

Comment:
Please be careful with the Chinese doc encoding...
See this https://issues.apache.org/jira/projects/IOTDB/issues/IOTDB-1052?filter=allissues
<img width="835" alt="Screen Shot 2021-01-13 at 3 33 17 PM" src="https://user-images.githubusercontent.com/25913899/104420444-d6bb3780-55b4-11eb-9251-22f64948be4e.png">

Reply:
It's over, I'll pay attention next time